src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -197,28 +213,53 @@ private static String prepareArgv0(Map<String, String> additionalEnvironment) th
                          .collect(joining(" "));
     String invocation = env.isEmpty() ? command : env + " " + command;
     String launcherContent = String.format(launcherTemplate, invocation);
-    Path launcher = Files.createTempFile("jazzer-", launcherExtension, launcherScriptAttributes);
+
+    Path launcher;
+    if(!isAndroid()){
+      launcher = Files.createTempFile("jazzer-", launcherExtension, launcherScriptAttributes);
+    }
+    else {
+      launcher = Files.createTempFile(Paths.get("/data/local/tmp/"), "jazzer-", launcherExtension, launcherScriptAttributes);
```

Comment:
Is `Files.createTempFile` available on Android, but `Files.createTempDirectory` isn't? Similar to the rules_jni PR, can we assume that this directory is always correct and exists?

Reply:
and also yes Files.createTempFile does work on Android